src/js/components/utils/useFetchIssues.tsx

```diff
@@ -29,16 +31,18 @@ export default ({
         url: addUrlParams(baseUrl, {
           project: projectId,
           is_attached: isAttached,
+          search,
```

Comment:
```suggestion
          search: search || undefined,
```

Reply:
@jgerigmeyer the type checker doesn't like this as the type in `api.ts` is `number | string | boolean`?